src/functions/raisebox.js

```diff
@@ -17,31 +42,41 @@ defineFunction({
         argTypes: ["size", "hbox"],
         allowedInText: true,
     },
-    handler({parser}, args) {
+    handler({parser, funcName}, args) {
         const amount = assertNodeType(args[0], "size").value;
         const body = args[1];
         return {
             type: "raisebox",
             mode: parser.mode,
+            funcName,
             dy: amount,
             body,
         };
     },
-    htmlBuilder(group, options) {
-        const body = html.buildGroup(group.body, options);
-        const dy = calculateSize(group.dy, options);
-        return buildCommon.makeVList({
-            positionType: "shift",
-            positionData: -dy,
-            children: [{type: "elem", elem: body}],
-        }, options);
+    htmlBuilder,
+    mathmlBuilder,
+});
+
+defineFunction({
+    type: "raisebox",
+    names: ["\\raise", "\\lower"],
+    props: {
+        numArgs: 2,
+        greediness: 0, // Less than the greediness of \hbox.
+        argTypes: ["size", "hbox"],
+        allowedInText: true,
     },
-    mathmlBuilder(group, options) {
-        const node = new mathMLTree.MathNode(
-            "mpadded", [mml.buildGroup(group.body, options)]);
-        const dy = group.dy.number + group.dy.unit;
-        node.setAttribute("voffset", dy);
-        return node;
+    handler({parser, funcName}, args) {
+        const amount = assertNodeType(args[0], "size").value;
+        const body = args[1];
+        return {
+            type: "raisebox",
+            mode: parser.mode,
+            funcName,
+            dy: amount,
```

Comment:
If the following change is made here we won't have to repeat the logic in both the HTML and MathML builders.
```suggestion
            dy: funcName === "\\lower" ? -amount : amount,
```